src/components/Hero/Abilities.jsx

```diff
@@ -23,23 +23,18 @@ const AbilityItem = styled.div`
   padding-right: 4px;
 `;
 
-const renderAbilities = (abilities) => {
-  const mappedAbilities = abilities.map((ability, key) => {
-    const mappedAbility = (
-      <AbilityItem key={key}>
-        <Ability {...ability} abilityID={key} />
-      </AbilityItem>
-    );
-
-    return mappedAbility;
-  });
-
-  return mappedAbilities;
-};
+const renderAbilities = abilities => abilities.map(ability => (
+  <AbilityItem key={abilities.key}>
```

Comment:
is abilities.key even defined? even if it was defined wouldn't each AbilityItem component get assigned the same key? @howardchung ?

Reply:
yeah, that needs to be fixed also

Reply:
I was unsure about this one, reverting the mapping changes but it will still have a `react/no-array-index-key` warning. Unless it can be resolved, we may want to have ESLint ignore this section.